docs/en/latest/plugins/ua-restriction.md

```diff
@@ -30,7 +30,7 @@ description: This document contains information about the Apache APISIX ua-restr
 
 The `ua-restriction` Plugin allows you to restrict access to a Route or Service based on the `User-Agent` header with an `allowlist` and a `denylist`.
 
-A common scenario is to set crawler rules. `User-Agent` is the identity of the client when sending requests to the server, and the user can whitelist or blacklist some crawler request headers in the `ua-restriction` Plugin.
+A common scenario is to set crawler rules. `User-Agent` is the identity of the client when sending requests to the server, and the user can `allowlist` or `denylist` some crawler request headers in the `ua-restriction` Plugin.
```

Comment:
```suggestion
A common scenario is to set crawler rules. `User-Agent` is the identity of the client when sending requests to the server, and the user can allow or deny some crawler request headers in the `ua-restriction` Plugin.
```

Reply:
allowlist and denylist are APISIX terms. They are used to allow or deny request headers. So, this would be better.